src/components/PhotoTags.vue

```diff
@@ -41,6 +41,7 @@ export default {
     },
     searchByTagName(query) {
       this.$router.push({ name: 'browse-page', query: { q: query } });
+      this.$router.go(0);
```

Comment:
Seems to me a bit strange to add this. Are you sure this is necessary? It might be something else causing this bug

Reply:
[In other places](https://github.com/creativecommons/cccatalog-frontend/blob/master/src/components/SearchGridForm.vue#L122), we use the store to commit the `SET_QUERY` mutation. I think this is the way to go about fixing this bug here

Reply:
I have made the changes using SET_QUERY mutation. Thanks for guiding me in this :-)